index.html

```diff
@@ -98,9 +98,10 @@ <h1>I’m Abdelaziz, Glad to see you!</h1>
             </div>
         </section>
         <!--work section-->
-
+        <div class="overlay disable"></div>
+        <div class="cards"></div>
         
-            <section class="experience">
+            <!-- <section class="experience">
```

Comment:
- [ ] Please remove all commented code from your codebase because it's a bad practice to commit commented-out code. That's what version control is for. If you need that code back, just fetch it from the repo :smile: